tox.ini

```diff
@@ -58,7 +58,6 @@ deps =
     functests: -r requirements/functests.txt
     bddtests: -r requirements/bddtests.txt
     dockercompose: -r requirements/dockercompose.txt
-    {env:EXTRA_DEPS:}
```

Comment:
This was previously used to allow you to inject `pip-tools` into a venv by running `EXTRA_DEPS=pip-tools tox -e ...`. This is no longer needed now that we've just added `pip-tools` to the `*.in` files (which is much better because it's now managed by Dependabot).